### PR TITLE
refactor: migrate vcent

### PR DESCRIPTION
**Pull request by lelenei**
## Description

Closes mf-2851

## Type of change

- [ ] Documentation
- [ ] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

## Checklist

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
  - [ ] I have removed all in development `console.log`s
  - [ ] I have removed all commented code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have read [Internationalization Guide](https://github.com/DimensionDev/Maskbook/blob/develop/docs/i18n-guide.md) and moved text fields to the i18n JSON file.

If this PR depends on external APIs:

- [ ] I have configured those APIs with CORS headers to let extension requests get passed. 
  - chrome extension: `chrome-extension://[id]`
  - firefox extension: `moz-extension://[id]`
- [ ] I have delegated all web requests to the background service via the internal RPC bridge.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -142,10 +142,8 @@
       "@emotion/react@11.10.4": "patches/@emotion__react@11.10.4.patch",
       "@mui/base@5.0.0-alpha.100": "patches/@mui__base@5.0.0-alpha.100.patch",
       "react-devtools-inline@4.26.1": "patches/react-devtools-inline@4.26.1.patch",
-      "@protobufjs/inquire@1.1.0": "patches/@protobufjs__inquire@1.1.0.patch",
       "micromark@3.1.0": "patches/micromark@3.1.0.patch",
       "micromark-util-symbol@1.0.1": "patches/micromark-util-symbol@1.0.1.patch",
-      "@sentry/core@7.3.0": "patches/@sentry__core@7.3.0.patch",
```
Comment (guanbinrui) [inline package.json:148]: CC @Lanttcat

Comment (Lanttcat) [reply package.json:148]: I's OK. The root package.json have this dep.

File: .i18n-codegen.json [hunk 2]
```diff
@@ -409,6 +409,18 @@
...
             }
+        },
+        {
+            "input": "./packages/plugins/VCent/src/locales/en-US.json",
+            "output": "./packages/plugins/VCent/src/locales/i18n_generated",
+            "parser": { "type": "i18next", "contextSeparator": "$", "pluralSeparator": "_" },
+            "generator": {
+                "type": "i18next/react-hooks",
+                "hooks": "useI18N",
+                "namespace": "com.maskbook.tweet",
```
Comment (Lanttcat) [inline .i18n-codegen.json:420]: This namespace is "tweet"?

File: package.json (hunk 1 above)
Comment (Jack-Works) [inline package.json:148]: why remove these two patches?

**Pull request merged**